### ISSUE TITLE
Bold hashtags does not go normal after deleting the # char

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#166]
```
It's similar to previous issue #155

What steps will reproduce the problem?
1. Write a hashtag, say #abc
2. The text goes bold
3. Delete the #

What is the expected output? What do you see instead?
The text should return to normal format, but it keeps bold.

What version of the product are you using? On what operating system?
0.2.4 karmic
```

Original issue reported on code.google.com by `mich.ba...@gmail.com` on 30 Mar 2010 at 11:26

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 31 Mar 2010 at 9:01
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Issue 172 has been merged into this issue.
```

Original comment by `tro...@gmail.com` on 1 Apr 2010 at 5:09

**Comment by GoogleCodeExporter** [comment]
```
Same thing happens when using the @ symbol for mentions
```

Original comment by `acunadit...@gmail.com` on 1 Apr 2010 at 5:44

**Comment by GoogleCodeExporter** [comment]
```
Closed by d934354d46 revision.
```

Original comment by `tro...@gmail.com` on 1 Apr 2010 at 7:50
- Changed state: **Fixed**
